docs/references/react/use-organization-list.mdx

```diff
@@ -1,3 +1,267 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganizationlist
+title: useOrganizationList()
+description: Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
 ---
+
+import { Callout } from "nextra-theme-docs";
+import { Tables } from "@/components/Table";
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
```

Comment:
Imports are not needed - the components are rendered by the marketing repo anyways. This is just tech debt.

```suggestion
```